web-app/views/game.hbs

```diff
@@ -6,31 +6,25 @@
     <title>Hex</title>
     <link rel="stylesheet" type="text/css" href="/css/style.css" />
     <link rel="stylesheet" type="text/css" href="/css/game.css" />
+    <script src="https://cdn.tailwindcss.com"></script>
   </head>
   <body>
     <div class="gameboard">
       <div class="box-header">
         {{#areEqual game.state.winner null}}
           {{#areEqual readyToPlay true}}
-            <p>
-              Current player:
-              <span class="current-player" data-player="{{game.state.turn}}">{{game.state.turn}}</span>
-            </p>
+            {{#areEqual currentPlayerTurnToPlay true}}
+            <p><span class="current-player" data-player="{{game.state.turn}}">{{game.state.turn}}</span>, your move!</p>
+            {{/areEqual}}
             {{#areEqual currentPlayerTurnToPlay false}}
```

Comment:
Suggestion:

```suggestion
            {{else}}
```

(I believe this works event with a custom Handlebars helper 🙂 )